Pod/Classes/BONText.m

```diff
@@ -575,6 +575,15 @@ - (NSString *)debugString
     return [self debugStringIncludeImageAddresses:YES];
 }
 
+- (NSString *)debugStringLeftToRight
```

Comment:
I was on the fence about that; I suppose it would be nice to have for someone that wants to use it in unit tests. Only tradeoff is naming (ie. `debugStringLeftToRightIncludeImageAddresses`) could be ugly. Was also considering some sort of refactoring to use a single `debugString` with parameters for both `leftToRight` and `includeImageAddresses`.